webapp/src/components/team_sidebar/team_sidebar.test.tsx

```diff
@@ -40,15 +40,15 @@ const mockSlice = createSlice({
     initialState: {
         'plugins-com.github.manland.mattermost-plugin-gitlab': {
             connected: true,
-            username: '',
+            username: 'mattermost',
             clientId: '',
             lhsData: {
                 reviews: [],
                 yourAssignedPrs: [],
                 yourAssignedIssues: [],
                 todos: [],
             },
-            gitlabURL: '',
+            gitlabURL: 'https://gitlab.com/gitlab-org/gitlab.git',
```

Comment:
This would actually be the org URL that the user is connected to, and not a repo URL

```suggestion
            gitlabURL: 'https://gitlab.com/gitlab-org',
```